client/src/components/AdvisorReports/FilterableAdvisorReportsTable.js

```diff
@@ -0,0 +1,160 @@
+import React, { Component } from 'react'
+import OrganizationAdvisorsTable from 'components/AdvisorReports/OrganizationAdvisorsTable'
+import Toolbar from 'components/AdvisorReports/Toolbar'
+import _debounce from 'lodash.debounce'
+import moment from 'moment'
+
+import API from 'api'
+
+const DEFAULT_WEEKS_AGO = 3
+const advisorReportsQueryUrl = `/api/reports/insights/advisors` // ?weeksAgo=3 default set at 3 weeks ago
+
+class FilterableAdvisorReportsTable extends Component {
+    constructor(props) {
```

Comment:
I would add here the propTypes. That way you force certain property types and you also document the component better.